Content/1.1 - Transaction Review/transactionReview.json

```
@@ -18,6 +18,21 @@
 	"transactionReview_networkFee_heading": "Transaction Fee",
 	"transactionReview_networkFee_congestedText": "The network is currently congested. Add a tip to speed up your transfer.",
 	"transactionReview_networkFee_customizeButtonTitle": "Customize",
+	"transactionReview_networkFeeSheet_default_title": "Customize Fees",
+	"transactionReview_networkFeeSheet_advanced_title": "Advanced Customize Fees",
+	"transactionReview_networkFeeSheet_default_subtitle": "Choose what account to pay the transaction fee from, or add a “tip” to speed up your transaction if necessary.",
+	"transactionReview_networkFeeSheet_advanced_subtitle": "Fully customize fee payment for this transaction. Not recommended unless you are a developer or advanced user.",
+	"transactionReview_networkFeeSheet_payFeeFrom": "Pay fee from",
+	"transactionReview_networkFeeSheet_changeButtonTitle": "Change",
+	"transactionReview_networkFeeSheet_noneRequired": "None required",
+	"transactionReview_networkFeeSheet_noneDue": "None due",
+	"transactionReview_networkFeeSheet_selectFeePayerButtonTitle": "Select Fee Payer",
+	"transactionReview_networkFeeSheet_viewNormalModeButtonTitle": "View Normal Mode",
+	"transactionReview_networkFeeSheet_viewAdvancedModeButtonTitle": "View Advanced Mode",
+	"transactionReview_networkFeeSheet_networkFee": "Network Fee",
+	"transactionReview_networkFeeSheet_royaltyFees": "Royalty fees",
+	"transactionReview_networkFeeSheet_networkRoyaltyFeesFieldLabel": "XRD to Lock for Network and Loyalty Fees",
```

`Loyalty` is also nice, but we should have `Royalty` :)
```suggestion
	"transactionReview_networkFeeSheet_networkRoyaltyFeesFieldLabel": "XRD to Lock for Network and Royalty Fees",
```